### ISSUE TITLE
[AREA]: Process inbound goods

### ISSUE THREAD

**Issue by ngump** [microsoft/dynamics365patternspractices#102]
### Contact details

ngump@microsoft.com

### Organization type

Microsoft employee

### End-to-end business process

Inventory to deliver

### Specify the business process area name for the article.

Process inbound goods

### Enter any additional comments or information you want us to know about this business process area.

Looking for help on this business process area.

### Specify the date you expect the article to be completed and ready for review.

09/30/2023

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ngump** [comment]
@Rachel ***@***.***> or @Eva Dupont (BAP ***@***.***> to action the below request from Lou.

Thanks,
Nicole

________________________________

**Comment by dj3500** [comment]
It looks like you guys have added like 6700 people to watch this repository. Removing some would be good.

**Comment by VonOgre** [comment]
I'm pretty sure that there's some default configuration to auto-subscribe
everyone that's joined to the Microsoft organization when a new repo spins
up.

**Comment by edupont04** [comment]
I'll continue to investigate, but I recommend all 6000+ people to click the link to _unsubscribe_. Sadly, we're seeing a weak spot in user management on GitHub.

**Comment by arbyruns** [comment]
We might need to file https://aka.ms/reportitnow since this is going to my personal github\email account

**Comment by VonOgre** [comment]
FYI, the auto-watching is an account setting that can be disabled.
![image](https://github.com/microsoft/dynamics365patternspractices/assets/18561847/ed03b20e-6a85-4c23-bba4-366a448340e7)

And you can update your watched repos from the same page (see View Watched Repositories)
![image](https://github.com/microsoft/dynamics365patternspractices/assets/18561847/d91ff174-ab8a-44ff-b83e-45e9530c774a)

**Comment by vainolo** [comment]
[surprised]     Arieh Bibliowicz reacted to your message:
________________________________

**Comment by peterbom** [comment]
@VonOgre: If we're receiving notifications because we've been given push access to a repo, then wouldn't a more appropriate course of action be to revoke those permissions, rather than bulk unsubscribe us? I for one certainly shouldn't have push permission to this repo.

**Comment by edupont04** [comment]
@ktoliver Yes, that's right. The problem is that each person must choose that unscubscribe link in the emails and/or unwatch the repo. In both cases, they'll need to sign in with their GitHub account, which fails for some....
